PublicFolders/SourceSideValidations/Tests/Permission/Remove-InvalidPermission.ps1

```diff
@@ -54,7 +54,7 @@ function Remove-InvalidPermission {
                     ) {
                         if ($PSCmdlet.ShouldProcess("$($result.FolderIdentity)", "Remove $($perm.User.DisplayName)")) {
                             Write-Host "Removing $($perm.User.DisplayName) from folder $($result.FolderIdentity)"
-                            $perm | Remove-PublicFolderClientPermission -Confirm:$false
+                            Remove-PublicFolderClientPermission $result.FolderEntryId -User $perm.User.DisplayName -Confirm:$false
```

Comment:
@bill-long making a change with write, need to have testing team aware of and test out.